### PR TITLE
QA-2738: create repo and linkedFeatures contract

### PR DESCRIPTION
**Pull request by nelsonds7**
Consumer contract for linkedFeatures

### CODE REVIEW

File: contract-specs/backend/linkedFeatures.pact.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const { PactV3, MatchersV3 } = require("@pact-foundation/pact");
+const path = require("path");
+
+const { eachLike } = MatchersV3;
+
+const provider = new PactV3({
+  dir: path.resolve(process.cwd(), "pacts"),
+  consumer: "Lodgify app",
+  provider: "Lodgify global",
```
Comment (GilbertoGNG) [inline contract-specs/backend/linkedFeatures.pact.spec.js:13]: let's use the same name than the repo of the provider. "lodgify-global"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "npx jest"
+  },
+  "keywords": [],
+  "author": "lodgify/qa-automation",
+  "license": "ISC",
+  "dependencies": {
+    "@pact-foundation/pact": "^12.1.3",
+    "sh": "^0.0.3"
```
Comment (GilbertoGNG) [inline package.json:14]: This dependency is not necessary

File: pacts/Lodgify app-Lodgify global.json
Comment (GilbertoGNG) [inline pacts/Lodgify app-Lodgify global.json:1]: From requests above this file will be different, specially its name. Let's try to not leave spaces in names of directories and files

File: publish-pacts.sh
Comment (GilbertoGNG) [inline publish-pacts.sh:1]: Replace it for 

```
#!/bin/bash

export PACT_BROKER_BASE_URL="https://pact-broker.lodgify.net"

docker run --rm \
 --platform=linux/arm64 \
 --network="host" \
 -v ${PWD}:${PWD} \
 -e PACT_BROKER_BASE_URL \
 -e PACT_BROKER_USERNAME \
 -e PACT_BROKER_PASSWORD \
  pactfoundation/pact-cli:latest \
  publish \
  ${PWD}/pacts \
  --consumer-app-version 1.0.0 \
  ```

File: contract-specs/backend/linkedFeatures.pact.spec.js (hunk 1 above)
Comment (nelsonds7) [reply contract-specs/backend/linkedFeatures.pact.spec.js]: Done!

File: package.json (hunk 2 above)
Comment (nelsonds7) [reply package.json]: Removed sh dependency

File: pacts/Lodgify app-Lodgify global.json
Comment (nelsonds7) [reply pacts/Lodgify app-Lodgify global.json:1]: done

File: publish-pacts.sh
Comment (nelsonds7) [reply publish-pacts.sh:1]: done

**Review by GilbertoGNG — APPROVED** [review]

**Pull request merged**